pkg/resources/etcd/statefulset.go

```diff
@@ -253,6 +253,11 @@ func StatefulSetCreator(data etcdStatefulSetCreatorData, enableDataCorruptionChe
 					},
 				},
 			}
+
+			set.Spec.Template.Spec.Containers[0].VolumeMounts = append(set.Spec.Template.Spec.Containers[0].VolumeMounts, resources.GetHostCACertVolumeMounts()...)
+
+			set.Spec.Template.Spec.Tolerations = data.Cluster().Spec.ComponentsOverride.Etcd.Tolerations
```

Comment:
Ah, we need that in our fork because we run the etcd pods on special local storage nodes now which are tainted so nothing else is scheduled on them. So the etcd pods need to tolerate those taints. So yeah it feels a bit arbitrary to have tolerations overridable in the etcd settings only, but it's what we needed.